e2e/package.json

```diff
@@ -4,7 +4,8 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "test": "playwright test"
+    "test": "playwright test",
+    "test:debug": "playwright test --debug"
```

Comment:
Actually dont need to add this leh. Usually if I want to add flag I will just `npm run test -- --debug`

Reply:
true 🤔️ I'll remove this and add a link [to this](https://playwright.dev/docs/debug) in the README that more clearly illustrates the debugging options available